### PR TITLE
[Bug] Fix bug that select into outfile in parquet format may cause NPE

### LINKED ISSUE #6055: [Bug]Select outfile with parquet format may cause NPE or BE crash
**Describe the bug**

1. NPE

    using `select * from xxx outfile` will cause NPE when analyzing the statement.

2. BE crash

    select outfile with parquet format without `properties` clause will cause BE crash

### PR DESCRIPTION
**Pull request by morningman**
## Proposed changes

1. check the parquet schema property on FE side.
2. auto generate parquet schema if not specified.

## Types of changes

What types of changes does your code introduce to Doris?
_Put an `x` in the boxes that apply_

- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update (if none of the other choices apply)
- [ ] Code refactor (Modify the code structure, format the code, etc...)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [x] I have created an issue on (Fix #6055 ) and described the bug/feature there in detail
- [ ] Compiling and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] If these changes need document changes, I have updated the document
- [ ] Any dependent changes have been merged

## Further comments

If this is a relatively large or complex change, kick off the discussion at dev@doris.apache.org by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by xinghuayu007** [comment]
LGTM

File: fe/fe-core/src/main/java/org/apache/doris/analysis/OutFileClause.java [hunk 1]
```diff
@@ -167,11 +175,113 @@ public void analyze(Analyzer analyzer) throws AnalysisException {
...
+                    break;
+                case DOUBLE:
+                    if (!type.equals("double")) {
+                        throw new AnalysisException("project field type is DOUBLE, should use double, but the definition type of column "
+                                + i + " is " + type);
+                    }
+                    break;
+                case CHAR:
+                case VARCHAR:
+                case DECIMAL:
```
Comment (yangzhg) [inline fe/fe-core/src/main/java/org/apache/doris/analysis/OutFileClause.java:235]: remove DCIMAL

File: fe/fe-core/src/main/java/org/apache/doris/analysis/OutFileClause.java [hunk 2]
```diff
@@ -167,11 +175,113 @@ public void analyze(Analyzer analyzer) throws AnalysisException {
...
+                    break;
+                case FLOAT:
+                    column.add("float");
+                    break;
+                case DOUBLE:
+                    column.add("double");
+                    break;
+                case CHAR:
+                case VARCHAR:
+                case DECIMAL:
```
Comment (yangzhg) [inline fe/fe-core/src/main/java/org/apache/doris/analysis/OutFileClause.java:276]: remove DECIMAL

Comment (morningman) [reply fe/fe-core/src/main/java/org/apache/doris/analysis/OutFileClause.java]: done

**Review by yangzhg — APPROVED** [review]
LGTM

**Pull request merged**